appveyor.yml

```diff
@@ -52,6 +52,10 @@ build_script:
 
 environment:
   NOT_CRAN: true
+  # See SPARK-27848. Currently installing some dependent packagess causes
+  # "(converted from warning) unable to identify current timezone 'C':" for an unknown reason.
+  # This environment variable works around to test SparkR against a higher version.
```

Comment:
can't we just set/change the timezone?

Reply:
Seems like that doesn't work .. I don't know the root cause TBH